wordstats/language_data/T2L/Code/Cleaning_words.py

```diff
@@ -0,0 +1,72 @@
+from pathlib import Path
+
+FREQ_WORDS_PATH = ''
+
+DICTIONARY_PATH = ''
+
+CLEAN_WORDS_PATH = ''
+
+LANG = ''
+
+
+def choose_language(lang_code):
+    global LANG
+
+    if lang_code == 'es':
+        LANG = 'Spanish'
+    elif lang_code == 'nl':
+        LANG = 'Dutch'
+    elif lang_code == 'de':
+        LANG = 'German'
+
+
+def dict_to_txt(dic, path):
+    with open(path, 'w') as file:
+        for line in dic:
+            file.write(line + '\n')
+
+
+def txt_to_dict(path):
+    freq_position = 1
+    words2clean = dict()
+
+    with open(path, 'r') as f:
+        for word in f.readlines():
+            words2clean[word.split()[0]] = freq_position
+            freq_position += 1
+
+    return words2clean
+
+
+def main():
+    global DICTIONARY_PATH, FREQ_WORDS_PATH, CLEAN_WORDS_PATH
+
+    lang2clean = input('Please, enter the language code you want to obtain clean words:\n')
```

Comment:
could I specify this as a program argument on the command line? 
and if it is not specified, then only ask me to type it in.